web/client/selectors/mapinfo.js

```diff
@@ -32,7 +34,27 @@ const mapInfoRequestsSelector = state => get(state, "mapInfo.requests") || [];
  */
 const generalInfoFormatSelector = (state) => get(state, "mapInfo.infoFormat", "text/plain");
 
+const measureActiveSelector = (state) => get(state, "measurement.lineMeasureEnabled") || get(state, "measurement.areaMeasureEnabled") || get(state, "measurement.bearingMeasureEnabled");
+const drawSupportActiveSelector = (state) => {
+    const drawStatus = get(state, "draw.drawStatus", false);
+    return drawStatus && drawStatus !== 'clean' && drawStatus !== 'stop';
+};
+const gridEditingSelector = (state) => get(state, "featuregrid.mode") === 'EDIT';
```

Comment:
the feature grid mode selector already exists, please use that as part of this selector

Reply:
Ok I'll use that.